CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.7.2] - 2021-06-01
```

Comment:
Update this with the actual version (0.8.0) and the correct date once the other branch is merged into the release branch. I'm going to remove myself as reviewer until the status of this PR is not draft.